src/components/Layout/Layout.module.scss

```diff
@@ -0,0 +1,27 @@
+@import '~ethos-design-system/src/components/Media/Media.scss';
```

Comment:
Good catch @roblevintennis

Reply:
I think this is totally understandable (and took like 30 seconds to fix) because we're bouncing back and forth between the EDS and CMS so it's very easy to do this. Realistically, once we get a bit more stable on EDS these sorts of things will come up much less imo